static/fetch_lib.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# shellcheck disable=2034,2059
+true
+# see https://github.com/koalaman/shellcheck/wiki/Directive
+
+# Functions
+lib_error_message() {
+    msg_box "You don't have internet and the local lib isn't available. Hence you cannot run this script."
+    exit 1
+}
+
+download_cache_lib() {
+    # Cache the lib for half an hour
+    if ! [ -f /var/scripts/lib.sh ] || test "$(find /var/scripts/lib.sh -mmin +30)"
+    then
+        rm -f /var/scripts/lib.sh
+        mkdir -p /var/scripts
+        if ! curl -so /var/scripts/lib.sh https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh
```

Comment:
question is, it is possible that the curl command succeeds, if the file already exists to which I want to curl another file?
If yes, I think this does already everything we want.

Reply:
Yes, it works. I've just tested this.